docs/source/dev_guide/algorithms/k_nearest_neighbors/k-nearest-neighbors-knn-classifier.rst

```diff
@@ -56,10 +53,23 @@ Given a positive integer parameter :math:`k` and a test observation
 #. Assigns the class with the largest probability to the test
    observation :math:`x_0`
 
-|product| version of the kNN algorithm uses the PANDA algorithm
-[Patwary2016]
-that uses a space-partitioning data structure known as K-D tree. Each
-non-leaf node of a tree contains the identifier of a feature along
+The library provides kNN classification with the following methods:
+
+- K-D tree
```

Comment:
K-D tree method is only supported on CPU

Reply:
fixed